src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/impl/CcdBundleStitchingService.java

```diff
@@ -9,44 +9,49 @@
 import pl.touk.throwing.ThrowingFunction;
 import uk.gov.hmcts.reform.em.orchestrator.service.CcdCaseUpdater;
 import uk.gov.hmcts.reform.em.orchestrator.service.dto.BundleDTO;
+import uk.gov.hmcts.reform.em.orchestrator.stitching.StitchingService;
+import uk.gov.hmcts.reform.em.orchestrator.stitching.StitchingServiceException;
 
 import java.util.List;
 import java.util.Spliterator;
 import java.util.Spliterators;
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
+import static pl.touk.throwing.ThrowingFunction.unchecked;
+
 @Service
 @Transactional
 public class CcdBundleStitchingService implements CcdCaseUpdater {
 
     private final ObjectMapper mapper = new ObjectMapper();
+    private final StitchingService stitchingService;
 
-//    public CcdBundleStitchingService(DocumentTaskItemProcessor documentTaskItemProcessor, BundleMapper bundleMapper) {
-//        this.documentTaskItemProcessor = documentTaskItemProcessor;
-//        this.bundleMapper = bundleMapper;
-//    }
+    public CcdBundleStitchingService(StitchingService stitchingService) {
+        this.stitchingService = stitchingService;
+    }
 
     @Override
     public void updateCase(JsonNode bundleData, String jwt) {
+        ArrayNode bundles = castJsonDataToJsonArray(bundleData);
```

Comment:
Pawel and I have changed this method to also take caseData. This is needed for populating new bundles, since we need caseData so that we can find all of the documents in there. Shall we just deal with the merge conflict when we merge the new-bundle-controller branch?

Reply:
yes